packages/node-sdk/paima-utils-backend/src/cde-access-internals.ts

```diff
@@ -190,3 +191,16 @@ export async function internalGetCardanoAddressDelegation(
 
   return results[0].pool;
 }
+
+export async function internalGetCardanoAssetUtxos(
+  readonlyDBConn: Pool,
+  address: string,
+  cip14Fingerprint: string
```

Comment:
Can we add an option to query by policy ID as well? Tracking a specific asset is useful for fungible tokens, and policy IDs useful for NFTs

We could make these both the same CDE (called something like delayed asset or delayed UTXO), or if it's easier to split them we can have the asset one be delayed token and the policy ID be delayed NFT

Reply:
I guess then we need to also be able to ask carp for the utxos by policy id, then. Otherwise you would have to specify in the cde config all of the fingerprints for that policy id. This is doable though, but I need to add it to carp first.

Reply:
Yeah it requires a change in Carp, but it's just a new query on the same data so it should be a fairly small change

Reply:
I think it's easier to keep them together, but I'm not sure about the API. Right now it can be configured like this:

```
extensions:
  - name: "assets1"
    type: cardano-delayed-nft
    fingerprints:
      - "asset1c43p68zwjezc7f6w4w9qkhkwv9ppwz0f7c3amw"
    policyIds:
      - "919d4c2c9455016289341b1a14dedf697687af31751170d56a31466e"
    startSlot: 22505578
```

Where both `fingerprings` and `policyIds` are optional (so one or the other can be used).

And I split the utils in `getCardanoAssetUtxosByFingerprint` and `getCardanoAssetUtxosByPolicyId`.

I could potentially rename the CDE to delayed utxo, but I'm not sure if mixing and matching fingerprints with policyids is a good idea or not. It may be better to have two different configs, and just use the same code for both?